src/App.css

```diff
@@ -8,27 +13,218 @@
 }
 
 .header {
+  border: 2px solid #fafafa;
   display: flex;
   justify-content: space-between;
-  margin: 0 5rem;
   align-items: center;
+  height: 3.5rem;
+  background-color: #ffff;
+  margin: 0 2rem;
+}
+
+.header1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #0290ff;
```

Comment:
- [ ] Hello @Santosh-Konappanavar, I notice repetition of `display: flex;` in all your container, One golden tip in styling is that we can create a class with any styling that is repeating and then give that class to any element we want. for example I can write
```CSS
.flex {
display: flex;
}
```
and then give `flex` class to any container I want. you can apply this rule to save a lot of lines in your CSS.

Reply:
Hello, **@brainconnect93**  Thanks for your kind suggestion I made changes as mentioned above.